helm/prometheus-rules/templates/alerting-rules/inhibit.management-cluster.rules.yml

```diff
@@ -80,7 +80,7 @@ spec:
     - alert: InhibitionClusterNodePoolsNotReady
       annotations:
         description: '{{`Cluster {{ $labels.cluster_id }} node pools are not ready. Either they have been scaled down to 0 or they are not up yet.`}}'
-      expr: cluster_operator_node_pool_desired_workers == 0 and cluster_operator_node_pool_ready_workers == 0
+      expr: cluster_operator_node_pool_desired_workers == 0 and cluster_operator_node_pool_ready_workers == 0 or capi_machinepool_status_condition{type="Ready", status="False"} == 1
```

Comment:
Does this now need to be in parenthesis? I'm not sure what the order of operations is for promql here.

Reply:
Good catch; looking at [prometheus docs, or seems to have the lowest priority](https://prometheus.io/docs/prometheus/latest/querying/operators/#binary-operator-precedence), but still, with parenthesis it's clearer for everyone :)
Thanks, added them in the last commit!